site/content/api/Tenant-API.md

```diff
@@ -436,3 +446,14 @@ The table below contains the properties which are used to configure a tenant's r
 | Name                     | Mandatory | JSON Type     | Default Value | Description |
 | :------------------------| :-------: | :------------ | :------------ | :---------- |
 | *max-connections*        | *no*      | *number*      | `-1`          | The maximum number of concurrent connections allowed from devices of this tenant. The default value `-1` indicates that no limit is set. |
+| *data-volume*            | *no*      | *object*      | `-`           | The maximum data volume allowed for the given tenant. Refer to  [Data Volume Configuration Format]({{< relref "#data-volume-configuration-format" >}}) for details.|
+
+### Data Volume Configuration Format
+
+The table below contains the properties which are used to configure a tenant's data volume limit:
+
+| Name                     | Mandatory | JSON Type     | Default Value | Description |
+| :------------------------| :-------: | :------------ | :------------ | :---------- |
+| *max-bytes*              | *no*      | *number*      | `-1`          | The maximum number of bytes allowed for the tenant for the given period defined by *period-in-days*. The default value `-1` indicates that no limit is set. |
+| *period-in-days*         | *no*      | *number*      | `30`          | The number of days for which the data usage is to be calculated. The default value  is `30` days.|
+| *effective-since*        | *yes*     | *string*      | `-`              | The date on which the data volume limit came into effect. The value MUST be an [ISO 8601 compliant](https://en.wikipedia.org/wiki/ISO_8601) without an offset with format `YYYY-MM-DD`.
```

Comment:
Is there a particular reason why we wouldn't allow using an offset?

Reply:
There is no specific reason and I don't see any disadvantages allowing it. To allow using an offset require some changes in the `PrometheusBasedResourceLimitChecks`.

Reply:
This shoudl actually be a localizable date/time, shouldn't it?

Reply:
I thought of `combined date and time representation in extended format` as defined for `not-before` and `not-after` in [Secrets Format](https://www.eclipse.org/hono/api/credentials-api/#secrets-format) which also supports various _zone offsets_.

Reply:
I have created a PR #1261 for the above.